### DISCUSSION TITLE
Disable winrm after Build

### DISCUSSION THREAD

**Discussion by jbpoint** [vmware/packer-examples-for-vsphere#147] [Help] [answered]
Hello,

building windows templates works like charm. 
But can you please advice, how to configure that winrm gets disabled after successful build?

**Comment by tenthirtyam** [comment, marked as answer]
You could include the commands in the guest customization when performing it via vSphere directly or in the [run_once_command_list](https://registry.terraform.io/providers/hashicorp/vsphere/latest/docs/resources/virtual_machine#run_once_command_list) of the Windows customization options if deploying via Terraform.

Ryan

**Reply by jbpoint** [reply]
Where exactly? I build the templates via vsphere and your repo.

**Reply by tenthirtyam** [reply]
Refer to the [Create a Customization Specification](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-CAEB6A70-D1CF-446E-BC64-EC42CDB47117.html) for Windows in the vSphere product documentation. Specifically point 10 for the optional `RunOnce` commands.

Ryan

**Reply by tenthirtyam** [reply]
Enhancement on the backlog; https://github.com/vmware-samples/packer-examples-for-vsphere/issues/265
